ibm/service/iamaccessgroup/resource_ibm_iam_access_group_account_settings.go

```diff
@@ -0,0 +1,81 @@
+package iamaccessgroup
+
+import (
+	"context"
+	"fmt"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceIBMIAMAccessGroupAccountSettings() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourceIBMIAMAccessGroupAccountSettingGet,
+		UpdateContext: resourceIBMIAMAccessGroupAccountSettingSet,
+		CreateContext: resourceIBMIAMAccessGroupAccountSettingSet,
+		DeleteContext: resourceIBMIAMAccessGroupAccountSettingUnSet,
+		Importer:      &schema.ResourceImporter{},
+		Schema: map[string]*schema.Schema{
+			"public_access_enabled": {
+				Type:        schema.TypeBool,
```

Comment:
why is it optional?

Reply:
My bad, it has to be required attribute for now as it is the only attribute in settings. in future with multiple settings, it can be optional.